src/main/java/core/basesyntax/model/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.model;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int NUMBER_OF_COLORS = Color.values().length;
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        return Color.values()[random.nextInt(NUMBER_OF_COLORS)].getCode();
+    }
```

Comment:
```suggestion
    private Random random = new Random();

    public String getRandomColor() {
        return Color.values()[random.nextInt(Color.values().length)].getCode();
    }
```

i dont think we need to create separate parameter for that